.fixtures-puppet6.yml

```diff
@@ -0,0 +1,13 @@
+fixtures:
+  repositories:
+    apt:
+      repo: git://github.com/puppetlabs/puppetlabs-apt.git
+      ref: 4.1.0
```

Comment:
isnt this 5.0.1 ?

Reply:
Yep, copied .fixtures.yml before that PR was merged.  Fixing.